collect_app/src/androidTest/java/org/odk/collect/android/instrumented/utilities/ImageConverterTest.java

```diff
@@ -382,6 +383,51 @@ public void scaleImageToNewWidthTest() {
         assertEquals(1250, image.getHeight());
     }
 
+    @Test
+    public void keepExifTest1AfterScaleAndRotation() {
```

Comment:
Does this test work for you locally? I tried on various android versions and the result is:
```
junit.framework.ComparisonFailure: expected:<ApertureValue_value> but was:<null>
at junit.framework.Assert.assertEquals(Assert.java:85)
at junit.framework.Assert.assertEquals(Assert.java:91)
at org.odk.collect.android.instrumented.utilities.ImageConverterTest.keepExifTest1AfterScaleAndRotation(ImageConverterTest.java:426)
at java.lang.reflect.Method.invoke(Native Method)
at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
at androidx.test.internal.runner.junit4.statement.RunBefores.evaluate(RunBefores.java:80)
at org.odk.collect.android.support.ResetStateRule$ResetStateStatement.evaluate(ResetStateRule.java:71)
```

Reply:
Not only locally I ran it on firebase test lab and the result is the same.

Reply:
I'll take a look at it unless you are already fixing the test.

Reply:
Ok I pushed one commit to your branch with fixes.